core-security/implementation/pom.xml

```diff
@@ -59,6 +59,11 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
+            <groupId>org.wildfly.common</groupId>
```

Comment:
@darranl do you see any issue with adding this dependency?

Reply:
I think the dependency is find but it looks like the module does not have the dependency so it is odd this is not resulting in an error.

Reply:
Actually there are a number of test failures, I would expect this module to also need the dependency adding:

https://github.com/wildfly/wildfly-core/blob/master/core-feature-pack/common/src/main/resources/modules/system/layers/base/org/jboss/as/core-security/main/module.xml